spec/lib/mercury/cps_spec.rb

```diff
@@ -10,40 +10,40 @@
 
   describe '::lift' do
     it 'CPS-transforms a non-CPS proc' do
-      expect(Cps.lift{rand}.run).to be_a Numeric
+      expect(Cps.lift { rand }.run).to be_a Numeric
     end
     it 'raises an error if the block returns a Cps object' do
-      expect{lift{lift{'cps'}}.run}.to raise_error /returned a Cps/
+      expect { lift { lift { 'cps' } }.run }.to raise_error /returned a Cps/
```

Comment:
I would prefer that for RSpec, no space is required between the `expect` and the first `{`.

I could even suggest that the `expect`s `{`, and `}` do not require an inner space. 

```ruby
 expect{lift { lift { 'cps' } }.run}.to raise_error /returned a Cps/